Source/Projections/Builder/UnregisteredProjections.cs

```diff
@@ -82,15 +96,28 @@ void AddToContainer(TenantId tenantId, IServiceCollection serviceCollection)
     {
         foreach (var projection in Values)
         {
+            var readModelType = projection.ProjectionType;
             serviceCollection.AddSingleton(
-                typeof(IProjectionOf<>).MakeGenericType(projection.ProjectionType),
-                serviceProvider => GetOfMethodForReadModel(projection.ProjectionType).Invoke(
+                typeof(IProjectionOf<>).MakeGenericType(readModelType),
+                serviceProvider => GetOfMethodForReadModel(readModelType).Invoke(
                     serviceProvider.GetRequiredService<IProjectionStore>(),
                     new object[]
                     {
                         projection.Identifier,
                         projection.ScopeId
                     }));
+            if (projection.Copies.MongoDB.ShouldCopy)
+            {
+                serviceCollection.AddSingleton(
+                    typeof(IMongoCollection<>).MakeGenericType(readModelType),
+                    serviceProvider => GetCollectionMethodForReadModel(readModelType).Invoke(
+                        serviceProvider.GetRequiredService<IMongoDatabase>(),
+                        new object[]
+                        {
+                            MongoDBCopyCollectionName.GetFrom(readModelType).Value,
+                            new MongoCollectionSettings()
```

Comment:
I think we should pass in `null` here so that the client uses the IMongoDatabase settings.
```suggestion
                            null
```